cmd/devguard/api/api.go

```diff
@@ -497,12 +499,16 @@ func Start(db core.DB) {
 
 	projectRouter.PUT("/members/:userId/", projectController.ChangeRole, projectScopedRBAC(accesscontrol.ObjectProject, accesscontrol.ActionUpdate))
 
+	//Api functions for interacting with an asset inside a project  ->  .../projects/<project-name>/assets/<asset-name>/...
 	assetRouter := projectRouter.Group("/assets/:assetSlug", projectScopedRBAC(accesscontrol.ObjectAsset, accesscontrol.ActionRead), assetMiddleware(assetRepository))
 	assetRouter.GET("/", assetController.Read)
 	assetRouter.DELETE("/", assetController.Delete, projectScopedRBAC(accesscontrol.ObjectAsset, accesscontrol.ActionDelete))
 
 	assetRouter.GET("/refs/", assetVersionController.GetAssetVersionsByAssetID)
 
+	//Api to scan manually using an uploaded SBOM provided by the user
+	assetRouter.POST("/sbom-manual-scan/", scanController.ManualSbomScan)
```

Comment:
```suggestion
	assetRouter.POST("/sbom", scanController.ManualSbomScan)
```

Lets rename that. Since the backend does not know, that the sbom was manually uploaded, i think removing this suffix makes sense.